features/fixtures/.ipynb_checkpoints/bakingtesttransform-checkpoint.py

```diff
@@ -0,0 +1,70 @@
+# ---
```

Comment:
not sure what this is, do we still need it?

Reply:
Leftover from testing the homemade dataset using jupyter lab. We don't need it so has been removed.